resources/assets/js/components/JobReview/JobReview.tsx

```diff
@@ -241,18 +241,8 @@ const sectionTitle = (title: string): React.ReactElement => {
 const languageRequirementIcons = (
   languageRequirementId: number,
 ): React.ReactElement => {
-  const enIcon = (
-    <img
-      src="/images/icon_english_requirement.svg"
-      alt="English language requirement icon."
-    />
-  );
-  const frIcon = (
-    <img
-      src="/images/icon_french_requirement.svg"
-      alt="French language requirement icon."
-    />
-  );
+  const enIcon = <img src="/images/icon_english_requirement.svg" alt="" />;
+  const frIcon = <img src="/images/icon_french_requirement.svg" alt="" />;
```

Comment:
Do we not need alt translations for these?

Reply:
If the image is purely decorative, empty alt attributes are the way to go. These icons have the full language spelled out beside them.

If the image is conveying information, you use alt attributes to capture it.